msg/UAVWaypointCommand.msg

```diff
@@ -0,0 +1,4 @@
+builtin_interfaces/Time time_stamp # Timestamp from when the message was send
+string sender_id # Name of the node that send this command
+Waypoint waypoint # Waypoint to fly too
+float32 speed_m_s 0 # Speed to use when flying to waypoint
```

Comment:
Could you clarify how to send this to the UAV via mavlink?

Reply:
How this message will be send to the UAV depends on the type of action. A RTH, land and takeoff will be done through a simple `mavsdk::Action` and the waypoint will be done through a `mavsdk::Mission`